src/ILLink.CodeFix/RequiresUnreferencedCodeCodeFixProvider.cs

```diff
@@ -32,15 +31,13 @@ public class RequiresUnreferencedCodeCodeFixProvider : BaseAttributeCodeFixProvi
 
 		public sealed override Task RegisterCodeFixesAsync (CodeFixContext context) => BaseRegisterCodeFixesAsync (context);
 
-		protected override SyntaxNode[] GetAttributeArguments (SemanticModel semanticModel, SyntaxNode targetNode, CSharpSyntaxNode containingDecl, SyntaxGenerator generator, Diagnostic diagnostic)
+		protected override SyntaxNode[] GetAttributeArguments (SyntaxGenerator syntaxGenerator, ISymbol attributableSymbol, ISymbol targetSymbol, Diagnostic diagnostic)
```

Comment:
I think you can minimize changes to this API by moving `SyntaxGenerator` back to being the second-to-last parameter